spec/controllers/migrations/external_controller_spec.rb

```diff
@@ -120,13 +120,13 @@
     context 'when decryption passes' do
       describe 'when decrypted params encapsulation is conform and data is valid' do
         context 'when external_id isn\'t present' do
-          it 'returns :created, creates a user and sends a reset email' do
+          it 'returns :created, creates a user' do
             encrypted_params = encrypt_params({ user: valid_user_params }, expires_in: 10.seconds)
 
             expect_any_instance_of(described_class).to receive(:generate_secure_pwd).and_call_original
             expect { post :create_user, params: { v2: { encrypted_params: } } }.to change(User, :count).from(0).to(1)
-            expect(ActionMailer::MailDeliveryJob).to have_been_enqueued.at(:no_wait).exactly(:once).with('UserMailer', 'reset_password_email',
-                                                                                                         'deliver_now', Hash)
+            expect(ActionMailer::MailDeliveryJob).not_to have_been_enqueued
```

Comment:
This is added to ensure that we don't mistakenly send any emails for this sensitive process,